scripts/solve.js

```diff
@@ -1,7 +1,119 @@
+const CODE_AREA_ID      = 'code-area';
+const CODE_OUTPUT_ID    = 'code-output';
+const RUN_BUTTON_ID     = 'run-button';
+const STOP_BUTTON_ID    = 'stop-button';
+const HIDDEN_ATTRIBUTE  = 'hidden';
+const NEWLINE           = '\n';
+
+var codeMirror;
+var outputArea;
+var runButton;
+var stopButton;
+var keepRunningCode;
```

Comment:
nit: use let or const ([style guide](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let))

Reply:
Thanks for the catch! Fixed it :)